project-koku/koku
# AWS Report filter and group by tags

Enabling filter and group_by tags. Behavior matches OCP report tag queries.

**Testing**
1. Add OCP and AWS providers with tags configured
2. Use the /api/v1/tags API to get sample tag key and values for OCP and AWS
3. Issue an AWS report query filtering on tag
4. Issue an AWS report query grouping on tag
5. Sanity check OCP report queries filter and group_by tags
6. Attempt AWS query for incorrect tag key and verify 400 is thrown


**Testing Results**
[koku_tag_aws_report_ut.txt](https://github.com/project-koku/koku/files/2771430/koku_tag_aws_report_ut.txt)

Closes #514 
Closes #515
